com.ibm.wala.core.tests/build.gradle

```diff
@@ -44,17 +44,13 @@ tasks.named('test') {
 	testLogging {
 		exceptionFormat = 'full'
 	}
-	// temporarily turn off some tests on JDK 11
-	if (JavaVersion.current() == JavaVersion.VERSION_11) {
-		exclude '**/callGraph/CallGraphTest.class'
-		exclude '**/callGraph/Java7CallGraphTest.class'
-		exclude '**/callGraph/CloneTest.class'
-		exclude '**/callGraph/DebuggingBitsetCallGraphTest.class'
-		exclude '**/callGraph/KawaCallGraphTest.class'
-		exclude '**/exceptionpruning/ExceptionAnalysisTest.class'
-		exclude '**/exceptionpruning/ExceptionAnalysis2EdgeFilterTest.class'
-		exclude '**/cha/LibraryVersionTest.class'
-		exclude '**/ir/TypeAnnotationTest.class'
+	// temporarily turn off some tests on JDK 11+
+	switch (JavaVersion.current()) {
```

Comment:
Why use a `switch` here as opposed to just an `if`?

Reply:
Over-engineered remnant from when I thought I would need more cases. Replaced with equivalent `if` in 43052fb065087b2510b584552afcaff124e47756. Thank you for your attentive reviewing!